### PR TITLE
User can view their entries - Sina Daryaei

### PR DESCRIPTION
**Pull request by sinadaryaei**
https://github.com/sinadaryaei/code-journal-/assets/125223265/911d992b-b370-4524-acc0-4c8fb08facd7

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -20,8 +21,107 @@ $form.addEventListener('submit', event => {
...
+  if (data.entries.length === 0) {
+    noEntriesText.classList.remove('hidden');
+  } else {
+    noEntriesText.classList.add('hidden');
+  }
+}
+
+function viewSwap(viewName) {
+  const entryformView = document.querySelector("[data-view='entry-form']");
+  const entriesView = document.querySelector("[data-view='entries']");
```
Comment (shawnkost) [inline js/main.js:103]: Move these outside the function. By having them inside of here, we'd have to traverse the DOM over and over every single time this function gets called. If we move the `querySelector`s outside the function to the global space, we can just do the search one and always have a reference to them

File: js/main.js [hunk 2]
```diff
@@ -20,8 +21,107 @@ $form.addEventListener('submit', event => {
...
+    entriesView.classList.remove('hidden');
+    entryformView.classList.add('hidden');
+  }
+  data.view = viewName;
+}
+
+const viewEntriesAnchor = document.querySelector('#view-entries');
+
+viewEntriesAnchor.addEventListener('click', function (event) {
+  event.preventDefault();
```
Comment (shawnkost) [inline js/main.js:118]: Not needed

File: js/main.js [hunk 3]
```diff
@@ -20,8 +21,107 @@ $form.addEventListener('submit', event => {
...
+  viewSwap('entries');
+});
+
+const newEntryButton = document.querySelector('#new-entry-button');
+
+newEntryButton.addEventListener('click', function (event) {
+  event.preventDefault();
```
Comment (shawnkost) [inline js/main.js:125]: Not needed

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes requested here Sina!

File: js/main.js (hunk 2 above)
Comment (sinadaryaei) [reply js/main.js:118]: Can you please specify which lines?

File: js/main.js (hunk 3 above)
Comment (sinadaryaei) [reply js/main.js:125]: Can you please specify which lines?

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**